### ISSUE TITLE
fix header to the  top of the page

### ISSUE THREAD

**Issue by JRussellHuffman** [Qiskit/qiskit.org#869]
When the user scrolls, the header stays fixed to the top of the browser window so that the menu options are still accessible even when the user has scrolled down the page.

**Comment by y4izus** [comment]
The `type:enhancement` label is going to [disappear](https://github.com/Qiskit/qiskit.org/issues/1041) . This enhancement is not related to the code quality but with a new feature, that's why I added the `type: feature request` label

**Comment by y4izus** [comment]
When implementing this issue, have in mind that the [menu on documentation page](https://github.com/Qiskit/qiskit_sphinx_theme/pull/22) should be modify too

**Comment by delapuente** [comment]
@JRussellHuffman is this still valid? Also, when talking about header, are we referring to the top menu?

**Comment by JRussellHuffman** [comment]
Yes, referring to the top menu. And yes, I would still like to see this implemented. Especially because the textbook pages can be so long.
